### PR TITLE
[doc] scheduler_perf: describe suite configuration in more detail

### LINKED ISSUE #86387: Document how to add/modify/run new scheduler benchmark tests
**What would you like to be added**:
We are making a series of enhancements #81719  to scheduler benchmarks. Once we are done, we should update the README.md.

Ref #86160, #85861

**Why is this needed**:

/sig scheduling

cc @notpad @ingvagabund

### PR DESCRIPTION
**Pull request by ingvagabund**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
The configuration file was design as a yaml file on purpose.
To easily extend the test cases without a need to modify
the testing binary. Also, it's possible to extend the configuration
itself to enrich individual test cases.

**Which issue(s) this PR fixes**:

Fixes: https://github.com/kubernetes/kubernetes/issues/86387

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by liu-cong** [comment]
cc @ahg-g

File: test/integration/scheduler_perf/README.md [hunk 1]
```diff
@@ -68,6 +68,25 @@ To produce a cpu profile:
 make test-integration WHAT=./test/integration/scheduler_perf KUBE_TIMEOUT="-timeout=3600s" KUBE_TEST_VMODULE="''" KUBE_TEST_ARGS="-alsologtostderr=false -logtostderr=false -run=^$$ -benchtime=1ns -bench=BenchmarkPerfScheduling -cpuprofile ~/cpu-profile.out"
 ```
 
+### How to configure bechmark tests
+
+Configuration file located under config/performance-config.yaml contains a list of templates.
+Each template allows to set:
+- node manifest
+- manifests for an initial and testing pod
```
Comment (liu-cong) [inline test/integration/scheduler_perf/README.md]: nit: `an initial and testing pod` -> `initial and testing pods`

File: test/integration/scheduler_perf/README.md [hunk 2]
```diff
@@ -68,6 +68,25 @@ To produce a cpu profile:
...
+- number of nodes, number of initial and testing pods
+- templates for PVs and PVCs
+- feature gates
+
+See `simpleTestCases` data type implementation for available configuration for each template.
+
+Initial pods (optionally with PVs) create a state of a cluster before the scheduler performance measurement can begin.
```
Comment (liu-cong) [inline test/integration/scheduler_perf/README.md]: I think the `optionally with PVs` can be removed. Different test cases can optionally have their specific configurations, and PV is only one of those cases.

**Review by liu-cong — COMMENTED** [review]
LG overall. Just a couple of nits.

File: test/integration/scheduler_perf/README.md (hunk 1 above)
Comment (ingvagabund) [reply test/integration/scheduler_perf/README.md]: Done

File: test/integration/scheduler_perf/README.md (hunk 2 above)
Comment (ingvagabund) [reply test/integration/scheduler_perf/README.md]: Removed

File: test/integration/scheduler_perf/README.md [hunk 3]
```diff
@@ -68,6 +68,25 @@ To produce a cpu profile:
...
+- feature gates
+
+See `simpleTestCases` data type implementation for available configuration for each template.
+
+Initial pods create a state of a cluster before the scheduler performance measurement can begin.
+Testing pods are then subject to performance measurement.
+
+The configuration file under config/performance-config.yaml contains a default list of templates to cover
+various scenarios. In case you want to add your own, you can extend the list with new templates.
+It's also possible to extend `simpleTestCases`, resp. its underlying data types to enrich possible test cases.
```
Comment (ahg-g) [inline test/integration/scheduler_perf/README.md]: I don't understand this sentence, what is "resp."?

Comment (ingvagabund) [reply test/integration/scheduler_perf/README.md]: resp. = respectively

Comment (ingvagabund) [reply test/integration/scheduler_perf/README.md]: I rephrased the sentence.

**Comment by liu-cong** [comment]
/retest

@ahg-g

**Pull request merged**